himl/main.py

```diff
@@ -63,6 +96,8 @@ def get_parser(parser=None):
                             help='the working directory')
         parser.add_argument('--multi-line-string', action='store_true',
                             help='will overwrite the global yaml dumper to use block style')
+        parser.add_argument('--merge-list-strategy', dest='merge_list_strategy', nargs=2,
```

Comment:
let's define as `type= DefaultMergeStrategy, choices=list(DefaultMergeStrategy)`

Reply:
Done.

Reply:
also renamed the argument name to list-merge-strategy for consistency. 
Pushed latest change. Plz review.